modules/category/category.service.js

```diff
@@ -136,6 +136,7 @@ class CategoryService extends FilterHelper {
       const modelsFields = models.map(async model => ({
         name: model.name,
         _id: model._id,
+        translations_key: model.translations_key,
```

Comment:
use camelCase instead of snake_case. wait for hot fix

Reply:
fixed